src/main/java/io/xstefank/wildfly/bot/util/GithubProcessor.java

```diff
@@ -128,21 +112,27 @@ public void processNotifies(GHPullRequest pullRequest, GitHub gitHub, Set<String
                 LOG.infof("Pull request #%d - PR review requested from \"%s\"", pullRequest.getNumber(),
                         String.join(",", reviewers));
             } else {
-                try {
-                    List<GHUser> ghReviewers = reviewers.stream()
-                            .map(user -> {
-                                try {
-                                    return gitHub.getUser(user);
-                                } catch (IOException e) {
-                                    // This should not be thrown as these are valid people - Collaborators
-                                    LOG.errorf(e, "User %s was not found", user);
-                                    throw new RuntimeException("Unable to find user " + user, e);
-                                }
-                            })
-                            .toList();
-                    pullRequest.requestReviewers(ghReviewers);
-                } catch (HttpException | RuntimeException e) {
-                    LOG.errorf("The request has failed due to %s", e.getMessage());
+                List<String> failedReviewers = new ArrayList<>();
+                for (String requestedReviewer : reviewers) {
+                    try {
+                        GHUser ghUser = gitHub.getUser(requestedReviewer);
+                        pullRequest.requestReviewers(List.of(ghUser));
+                    } catch (HttpException | RuntimeException e) {
+                        LOG.errorf(
+                                "The request to getting GHUser or requesting the user as Pull Request reviewer has failed due to %s",
+                                e.getMessage());
+                        failedReviewers.add(requestedReviewer);
+                    }
+                }
+
+                if (!failedReviewers.isEmpty()) {
+                    LOG.infof("Bot can not request PR review from the following people: %s", failedReviewers);
```

Comment:
and this is info?

Reply:
It was rather meant as info to the mail, that is send next, so the it is clearer from the logs and you do not have to read it from the sent mail info log